### PR TITLE
Package TTT into a ruby gem

### PR DESCRIPTION
**Pull request by Middlebrooks314**
https://trello.com/c/3bjTbaaM

### CODE REVIEW

File: .rspec_status [hunk 1]
```diff
@@ -0,0 +1,78 @@
+example_id                          | status | run_time        |
```
Comment (cmvandrevala) [inline .rspec_status:1]: Should this file be in your .gitignore?

**Review by cmvandrevala — APPROVED** [review]
I left one comment on the PR, but otherwise it looks good to me

File: .rspec_status (hunk 1 above)
Comment (Middlebrooks314) [reply .rspec_status:1]: yes, I think I must have overwritten it with the merge. Fixed now

Comment (cmvandrevala) [reply .rspec_status:1]: You should still probably remove this file from the commit.

**Pull request merged**